operator/controllers/execution/scans/scan_reconciler.go

```diff
@@ -95,26 +94,31 @@ func (r *ScanReconciler) startScan(scan *executionv1.Scan) error {
 	scan.Status.RawResultType = scanType.Spec.ExtractResults.Type
 	scan.Status.RawResultFile = filepath.Base(scanType.Spec.ExtractResults.Location)
 
-	findingsDownloadURL, err := r.PresignedGetURL(scan.UID, "findings.json", 7*24*time.Hour)
+	urlExpirationDuration, err := util.GetUrlExpirationDuration(util.ScanController)
+	if err != nil {
+		r.Log.Error(err, "Failed to parse scan url expiration")
+	}
+
+	findingsDownloadURL, err := r.PresignedGetURL(scan.UID, "findings.json", urlExpirationDuration)
 	if err != nil {
 		r.Log.Error(err, "Could not get presigned url from s3 or compatible storage provider")
 		return err
 	}
 	scan.Status.FindingDownloadLink = findingsDownloadURL
-	rawResultDownloadURL, err := r.PresignedGetURL(scan.UID, scan.Status.RawResultFile, 7*24*time.Hour)
+	rawResultDownloadURL, err := r.PresignedGetURL(scan.UID, scan.Status.RawResultFile, urlExpirationDuration)
```

Comment:
I don't think these should be changed or if we do change them they should be separate config values.
These are meant to be longer living as they are attached to the scan for humans / external systems interfacing with the scans.